engine/src/main/java/org/entando/entando/web/group/GroupController.java

```diff
@@ -60,8 +75,11 @@ public void setGroupValidator(GroupValidator groupValidator) {
 
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<?> getGroups(RestListRequest requestList) {
+    public ResponseEntity<?> getGroups(RestListRequest requestList) throws JsonProcessingException {
+        this.getGroupValidator().validateRestListRequest(requestList);
         PagedMetadata<GroupDto> result = this.getGroupService().getGroups(requestList);
+        this.getGroupValidator().validateRestListResult(requestList, result);
+        logger.debug("Main Response -> " + new ObjectMapper().writeValueAsString(result));
```

Comment:
This should be parameterized so the string is only realized if debug is enabled.